core/templates/dev/head/pages/exploration_editor/ExplorationDataService.js

```diff
@@ -72,14 +75,9 @@ oppia.factory('ExplorationDataService', [
           // We can safely remove the locally saved draft copy if it was saved
           // to the backend.
           LocalStorageService.removeExplorationDraft(explorationId);
-          if (successCallback) {
-            successCallback(response);
-          }
-        }, function() {
-          if (errorCallback) {
-            errorCallback();
-          }
-        });
+          successCallback(response);
+        },
+        errorCallback());
```

Comment:
errorCallback, not errorCallback(). The argument is a function, not the result of the function's execution.

Reply:
Done.